tests/test_cte.py

```diff
@@ -711,3 +713,132 @@ def test_django52_ambiguous_column_names(self):
             ('venus', 22, "admin"),
             ('venus', 23, "admin"),
         ])
+
+    def test_django52_queryset_aggregates_klass_error(self):
```

Comment:
nitpick
```suggestion
    def test_django52_queryset_aggregates_class_error(self):
```

Reply:
This one was not a typo, the exception that was thrown was caused by Django accessing a `klass` variable which was unexpectedly `None`.